packages/cli/src/utils/sql.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Provides syntax highlighting for embedded SQL queries in template strings.
+ */
+export function sql(strings: TemplateStringsArray, ...values: string[]): string {
```

Comment:
1. where we actually need highlighting, we could use the `sql-highlight` package instead of implementing this ourselves
2. this seems to be only used in tests, no values are passed, and the usage doesn't seem like it benefits much from template strings. do we really need this?

Reply:
@despairblue introduced it for syntax highlighting reasons. I guess we could use a package also. 
we actually use the tag function with values on code we are working on for the data compaction (next step on the insights feature)